lib/check_data_points.rb

```diff
@@ -0,0 +1,9 @@
+class CheckDataPoints
+  def self.tipline_requests(team_id, date = nil)
+  	# date in format `2023-08-23`
+  	date = begin DateTime.parse(date) rescue nil end
+  	query = TiplineRequest.where(team_id: team_id)
+  	query = query.where(created_at: date.all_day) unless date.nil?
+  	query.count
```

Comment:
If the method receives an argument `group`, the count should be grouped by that time range. For example: If `grouped = 'daily'`, the return should be `query.group('DAY(created_at)').count`.